src/Chapter1.hs

```diff
@@ -449,7 +454,10 @@ Implement the function that takes an integer value and returns the next 'Int'.
   function body with the proper implementation.
 -}
 next :: Int -> Int
-next x = error "next: not implemented!"
+next x = x + 1
+{-
+greater than max -> minimum
```

Comment:
Good catch!
We will discuss this in future chapters 😄

Reply:
Looking forward to future chapters  🍡